### ISSUE TITLE
improve contract of Recontact "Me" red for A11y

### ISSUE THREAD

**Issue by trollepierre** [trollepierre/recontact_travel_blog#765]
Elements must have sufficient color contrast: Element has insufficient color contrast of 2.63 (foreground color: #f48024, background color: #ffffff, font size: 19.5pt (26px), font weight: bold). Expected contrast ratio of 3:1
https://www.recontact.me/:7:453

<span class="me" data-v-7a93515c="">Me</span>
